can-bus/json/GSM/GSM_rx.json

```
@@ -1,3 +1,5 @@
 {
-    "messages": []
+    "messages": [
+        "Debug_CanModes"
+    ]
 }
```

nit: missing blank line
```suggestion
}

```